src/tests/Register.test.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import Register from "../components/Register";
+import { BrowserRouter } from "react-router-dom";
+
+it("renders without crashing", () => {
```

Comment:
@aematei & @alinz22  This code should be run after you commit to the GIthub.  Look at the example here
https://github.com/luminosityleds/client/blob/master/.github/workflows/typescript_coverage.yml#L16

You would only need to run `npm run vitest` on the runner.

Reply:
Hey @pogi7 I'm not quite sure what you mean. We added this test to check that this page renders, and to get our coverage above the threshold, since adding a new page dropped coverage down significantly. Is there something else we should do to fix that?

Reply:
I believe this code is only run on commit.